test/test_framework_test.go

```diff
@@ -1230,7 +1249,7 @@ func TestUsingEnv(t *testing.T) {
 		}
 
 		contracts := map[string]common.Address{
-			"FooContract": {0, 0, 0, 0, 0, 0, 0, 5},
```

Comment:
Given that this had to be adjusted in many places: maybe introduce a constant like e.g. "`expectedBootstrappedContractCount`" and use it here and in the other places

Reply:
Good point :+1: 
I have added 3 variables to hold the first 3 account addresses in https://github.com/onflow/cadence-tools/pull/280/commits/a522bb10a915d9c3d80c2c8ea08b6c6bbe1fd978